src/components/modals/ActionScorer.tsx

```diff
@@ -275,14 +293,17 @@ class ActionScorer extends React.Component<Props, ComponentState> {
             haveEdited: false,
             newAction: null,
             cardViewerAction: null,
-            cardViewerShowOriginal: false
+            cardViewerShowOriginal: false,
+            isAlreadySelectedOpen: false
         };
         this.handleActionSelection = this.handleActionSelection.bind(this);
         this.handleDefaultSelection = this.handleDefaultSelection.bind(this);
         this.handleOpenActionModal = this.handleOpenActionModal.bind(this);
         this.renderItemColumn = this.renderItemColumn.bind(this);
         this.onColumnClick = this.onColumnClick.bind(this);
         this.focusPrimaryButton = this.focusPrimaryButton.bind(this);
+        this.showPopup = this.showPopup.bind(this);
```

Comment:
Would prefer var names to be more detailed in case we add other one in the future (i.e. "showAlreadySelectedPopup, etc